.docs/user-guide/troubleshooting.md

```diff
@@ -139,29 +167,40 @@ curl http://127.0.0.1:7979/services
 ```
 ## Some common errors
 
-General note: When running on a public cloud provider the error messages in the log resulting from calls to a cloud provider API are repeated from the cloud provider’s API. Sometimes these are not intuitive. For example a bad credential might result in an error message related to a non-existent object. Performing a google search in the context of your cloud provider, rather than REX-Ray can sometimes prove to be helpful. 
+General note: When running on a public cloud provider the error messages in 
+the log resulting from calls to a cloud provider API are repeated from the 
+cloud provider’s API. Sometimes these are not intuitive. For example a bad 
+credential might result in an error message related to a non-existent object. 
+Performing a google search in the context of your cloud provider, rather than 
+REX-Ray can sometimes prove to be helpful. 
 
-### Issue: Attempt to start REX-Ray service as a normal account, rather that root or other privileged:
-```
+#### Issue: Attempt to start REX-Ray service as a normal account, rather that root or other privileged:
+```sh
+$ rexray service start
 Failed to start rexray.service: Interactive authentication required.
 ```
 
-### Issue: Leave out service specification on command line, when multiple services are configured:
-`FATA[0000] http error                                    status=404`
+#### Issue: Leave out service specification on command line, when multiple services are configured:
```

Comment:
This is not same as a missing config, this is where a config has multiple services but you fail to earmark one of them in a command line operation, such as volume ls in this example.
.. and the example of a missing config file is already present (see line 189)